sway/commands/input/xkb_layout.c

```diff
@@ -1,7 +1,5 @@
 #define _POSIX_C_SOURCE 200809L
-#include "sway/config.h"
```

Comment:
`sway/config.h` should be kept. It'll compile without it since `sway/commands.h` includes it, but it's better to be explicit.